### ISSUE TITLE
The “Translate page…” panel item have missing icon in latest Nightly

### ISSUE THREAD

**Issue by Patricek0056** [black7375/Firefox-UI-Fix#709]
### Describe the bug

The "Translate page..." applications menu item have missing icon in latest Nightly 115.

Maybe it sounds like a fiction, but Yes! Firefox would have translator when Firefox 115 will be released:
- https://blog.nightly.mozilla.org/2023/06/01/firefox-translations-and-other-innovations-these-weeks-in-firefox-issue-139/

### Expected behavior

The “Translate page…” applications menu item should have an icon.

### Screenshots

![The Translate page  have missing icon](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/5e858739-09df-4705-8e91-da98d29482a6)

### OS

Windows 11

### Firefox Version

v115 Nightly

### Distribution

- [X] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
- [ ] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
- [ ] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)

### Theme

- [X] Light
- [X] Dark
- [ ] Alpenglow (System: Light)
- [ ] Alpenglow (System: Dark)
- [ ] System Default
- [ ] Others: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

### `user.js` setup

<details><summary><strong><code>user.js</code> setup</strong></summary><br>

```javascript
I don't know, because I have it installed through Windows Sandbox VM.
```
</details>

**Comment by Patricek0056** [comment]
There is a new icon in the address bar, which is part of this new translator. So, you should use that icon.

![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/2b9bf7e2-24de-417b-96b1-e38b1ebc73ea)

**Comment by Patricek0056** [comment]
It is fixed! Thank you for quick patch.
